### PR TITLE
Add arabic localization

### PR DESCRIPTION
**Pull request by ahmed-0011**

### CODE REVIEW

**Comment by gujjwal00** [comment]
@ahmed-0011 can you look at the build failure?

**Comment by ahmed-0011** [comment]
```bash
/home/runner/work/avnc/avnc/app/src/main/res/values-ar/strings.xml:108:4: Failed to flatten XML for resource 'tip_empty_server_list' with error: Invalid unicode escape sequence in string
"{str}"
/home/runner/work/avnc/avnc/app/src/main/res/values-ar/strings.xml:108:4: string/tip_empty_server_list does not contain a valid string resource.
```

I think there is a missing \ to escape one of the single quotes in the string. I will make a new commit and see if the issue will be resolved.

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -0,0 +1,233 @@
+<!--
+  ~ Copyright (c) 2020  Gaurav Ujjwal.
```
Comment (gujjwal00) [inline app/src/main/res/values-ar/strings.xml:2]: You can change this to your name if you want

File: app/src/main/res/values-ar/strings.xml [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+  ~
+  ~ SPDX-License-Identifier:  GPL-3.0-or-later
+  ~
+  ~ See COPYING.txt for more details.
+  -->
+
+<resources>
+    <string name="app_name" translatable="false">AVNC</string>
```
Comment (gujjwal00) [inline app/src/main/res/values-ar/strings.xml:10]:
```suggestion
```

File: app/src/main/res/values-ar/strings.xml [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    <string name="pref_input">المدخل</string>
+    <string name="pref_input_summary">الإيماءات والفأرة والمفاتيح الافتراضية</string>
+    <string name="pref_servers">الخوادم</string>
+    <string name="pref_server_summary">تسجيل الدخول والاكتشاف</string>
+    <string name="pref_tools">الأدوات</string>
+    <string name="pref_tools_summary">الاستيراد/التصدير وتصحيح الأخطاء</string>
+    <string name="pref_experimental">تجريبي</string>
+    <string name="pref_experimental_summary">ميزات إضافية</string>
+
+    <string name="pref_gesture_action_none" translatable="false">@string/title_none</string>
```
Comment (gujjwal00) [inline app/src/main/res/values-ar/strings.xml:149]:
```suggestion
```

File: app/src/main/res/values-ar/strings.xml [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+
+    <string name="pref_theme">السمة</string>
+    <string name="pref_theme_option_system">النظام</string>
+    <string name="pref_theme_option_dark">داكن</string>
+    <string name="pref_theme_option_light">فاتح</string>
+    <string name="pref_prefer_advanced_editor">فتح خيارات الخادم المتقدمة مباشرةً</string>
+    <string name="pref_sort_server_list">ترتيب الخوادم المحفوظة حسب الاسم</string>
+    <string name="pref_gesture">الإيماءات</string>
+    <string name="pref_gesture_style">نمط الإيماءة</string>
+    <string name="pref_gesture_style_auto" translatable="false">@string/title_automatic</string>
```
Comment (gujjwal00) [inline app/src/main/res/values-ar/strings.xml:169]:
```suggestion
```

Comment (gujjwal00) [reply app/src/main/res/values-ar/strings.xml:169]: These non translatable strings should be removed.

**Pull request merged**